mindmap/static/js/src/mindmap.js

```diff
@@ -1,5 +1,13 @@
-function MindMapXBlock(runtime, element) {
-    var mind = {
+const { notify } = new window.StudioRuntime.v1();
+
+notify("save", {
+    state: "start",
+    message: "Saving...",
+});
+
+function MindMapXBlock(runtime, element, context) {
+
+    const baseMind = {
         "meta": {
             "name": "Mind Map",
             "author": "eduNEXT",
```

Comment:
can this be changed?

Reply:
Change baseMind?

Reply:
the author

Reply:
Maybe we could pass the student's name in the context?

Reply:
Yes! That's a good idea